src/urllib3/util/proxy.py

```diff
@@ -45,6 +45,8 @@ def create_proxy_ssl_context(
         ssl_version=resolve_ssl_version(ssl_version),
         cert_reqs=resolve_cert_reqs(cert_reqs),
     )
+    ssl_context.verify_mode = resolve_cert_reqs(cert_reqs)
```

Comment:
I think `create_urllib3_context` already sets `verify_mode`?

https://github.com/urllib3/urllib3/blob/1.26.x/src/urllib3/util/ssl_.py#L338

Reply:
Ah, yes you're right. let me remove that.